src/server/handlers/launchpad.js

```diff
@@ -122,107 +143,137 @@ const responseError = (res, error) => {
   }
 };
 
-const makeSnapName = url => {
-  return createHash('md5').update(url).digest('hex');
+const checkStatus = (response) => {
```

Comment:
If it's only for GH (as it seems from possible error responses) it would be good to name it to reflect that. So it's not used elsewhere as a generic 'checkStatus'.